pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -333,3 +333,31 @@ func addStandardNodeAttributes(attr map[string]*mpb.Attributes_AttributeValue, p
 		}
 	}
 }
+
+func serviceConfig(serviceHostname string, dest *model.ServiceInstance, config model.IstioConfigStore, disableCheck, disableReport bool) *mccpb.ServiceConfig {
+	sc := v1.ServiceConfig(serviceHostname, dest, config, disableCheck, disableReport)
```

Comment:
can you copy over the functions from that directory? I don't care about code duplication. We are going to be deleting that directory sometime or the other. So, lets not create dependencies

Reply:
I can, but I didn't create any dependencies. This simply wrapped the existing dependency. FWIW, it is used elsewhere in this same file directly.  Thoughts?

Reply:
went ahead and copied it over.

Reply:
if you could get rid of those as well, it would be great. I am trying to avoid referring to anything in the v1 package